pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -257,7 +259,7 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	// Return if this claim has been satisfied
-	if accountClaim.Spec.AccountLink != "" && accountClaim.Status.State == awsv1alpha1.ClaimStatusReady {
+	if accountClaim.Spec.AccountLink != "" && accountClaim.Status.State == awsv1alpha1.ClaimStatusReady && accountClaim.Spec.AccountOU != "" {
```

Comment:
Could we move this check into a function that returns a bool? Similar to what Christ recently did for the BYOC logic.

Reply:
:+1: